kumascript/src/environment.js

```diff
@@ -133,6 +138,7 @@ class Environment {
     globals.page = globals.Page = freeze(page);
     globals.env = globals.Env = freeze(env);
     globals.info = freeze(info);
+    globals.renderFromURL = renderFromURL;
```

Comment:
Since this function will receive URL's from the "wilds" of the content, it's important to clean the incoming URL's and maybe it's best to do that here?
```suggestion
    globals.renderFromURL = (url) => renderFromURL(info.getUriKey(url));
```